website/docs/framework-docs/guides/configuration.md

```diff
@@ -183,7 +183,7 @@ COMING SOON, STAY TUNED!
 | `read_timeout` | Read timeout in seconds for the API. | `number` | 5 |
 | `write_timeout` | Write timeout in seconds for the API. | `number` | 5 |
 | `logging` | Logging configuration. See [Logging Config below](#logging-config). | `object` |  |
-
+| `schema_path` | Path of schema file at location src/test/resources | `string` |  |
```

Comment:
Wrap `src/test/resources` in between a tick [<code>`</code>]

Reply:
Done